integration_tests/auction_test.go

```diff
@@ -160,8 +171,21 @@ func (s *IntegrationTestSuite) TestAuctionModule() {
 		s.Require().NoError(err)
 		s.T().Logf("Orchestrator 0 token balances after first bid %v", balanceRes.Balances)
 
-		s.Require().Equal(sdk.NewCoin("gravity0x3506424f91fd33084466f402d5d97f05f8e3b4af", sdk.NewInt(2500000000)), balanceRes.Balances[0])
-		s.Require().Equal(sdk.NewCoin("usomm", initialOrchBalanceRes.Balances[1].Amount.Sub(sdk.NewInt(5000000000))), balanceRes.Balances[2])
+		foundSomm, foundGravity := false, false
+		latestOrchSomm, latestOrchGravity := sdk.ZeroInt(), sdk.ZeroInt()
+		for _, balance := range balanceRes.Balances {
```

Comment:
Was the hardcoded assumption about balance positions wrong? Just curious as to what necessitated the fix.

Reply:
Yes, genesis state was changed for cellarfees testing that messed up auction test indeces.